optaplanner-core/src/main/java/org/optaplanner/core/api/score/Score.java

```diff
@@ -60,15 +63,22 @@
      * For example {@code -7init/0hard/-8soft} returns {@code 0hard/-8soft}.
      *
      * @return equal score except that {@link #getInitScore()} is {@code 0}.
+     * @deprecated Use {@link #withInitScore(int)} with 0 as the argument.
      */
-    Score_ toInitializedScore();
+    @Deprecated
+    default Score_ toInitializedScore() {
+        if (isSolutionInitialized()) {
+            return (Score_) this;
+        } else {
+            return withInitScore(0);
+        }
+    }
```

Comment:
I was able to replace methods on the individual `Score` implementations with this one.